src/main/java/address/storage/StorageManager.java

```diff
@@ -181,16 +181,20 @@ public static UserPrefs getUserPrefs(File prefsFile) {
     public void start() {
         logger.info("Starting storage manager.");
         loadDataFromFile(prefs.getSaveLocation());
-        raise(new SaveLocationChangedEvent(prefs.getSaveLocation()));
```

Comment:
This event call is to initialise the label at the footer on save file location. If this is removed, the label will be empty.